hpedockerplugin/volume_manager.py

```diff
@@ -777,7 +779,7 @@ def mount_volume(self, volname, vol_mount, mount_id):
 
         # Call OS Brick to connect volume
         try:
-            device_info = self._connector.\
+            device_info = self._connector. \
```

Comment:
is this space needed after _connector.     ?

Reply:
Looks like pycharm was at work and I didn't realize. Using '\' to continue statement on next line is generally discouraged. Removing it.